plugins/modules/rabbitmq_feature_flag.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Damian Dabrowski <damian@dabrowski.cloud>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: rabbitmq_feature_flag
+short_description: Enables feature flag
+description:
+  - Allows to enable specified feature flag
+author: "Damian Dabrowski (@damiandabrowski5)"
+options:
+  name:
+    description:
+      - Feature flag name
+    type: str
+    required: true
+  node:
+    description:
+      - erlang node name of the target rabbit node
+    type: str
+    required: false
+    default: rabbit
+'''
+
+EXAMPLES = '''
+# Enable the 'maintenance_mode_status' feature flag on 'rabbit@node-1'
+- community.rabbitmq.rabbitmq_feature_flag:
```

Comment:
```suggestion
- name: Enable the 'maintenance_mode_status' feature flag on 'rabbit@node-1'
  community.rabbitmq.rabbitmq_feature_flag:
```
This is a requirement. I fixed all the modules yesterday